src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
Ah, I see. But won't most (all?) Color3fPlugs have the default interpretation in practice anyway, so they'll be omitting it already?